pkg/eventfilter/attributes/filter.go

```diff
@@ -59,8 +60,10 @@ func (attrs attributesFilter) Filter(ctx context.Context, event cloudevents.Even
 	for k, v := range attrs {
```

Comment:
This entire method should be rewritten to me more performant. No need to explode the event for filters that do not care about most of the attributes.

Reply:
(this comment was not about this PR, but the code in general. I used it to make a new issue)